media/js/firefox/new.js

```diff
@@ -171,12 +171,41 @@
                     $thankYou.focus();
                 }, 500);
             }
+
+            // Added measurements fire only when the download is fired
+            spring_campaign_measurement();
         }
 
         function show_scene_anim(scene) {
             show_scene(scene, true);
         }
 
+        // Pixel to be removed on July 31st, 2015 (Bug 1168440)
+        function spring_campaign_measurement () {
+            var _dntStatus = navigator.doNotTrack || navigator.msDoNotTrack;
+            var fxMatch = navigator.userAgent.match(/Firefox\/(\d+)/);
+            var ieMatch = navigator.userAgent.match(/MSIE/);
+
+            if (fxMatch && Number(fxMatch[1]) < 32) {
+                // Can't say for sure if it is 1 or 0, due to Fx bug 887703
+                _dntStatus = (_dntStatus === 'yes') ? 'Unknown' : 'Unspecified';
+            } else if (ieMatch) {
```

Comment:
Since it's only IE10 on Windows 8 that has DNT enabled without explicitly being set by the user, we should only disregard DNT for that specific combo. Users on IE9 or IE10 on Windows 10 should have their DNT setting respected.